config/initializers/omniauth.rb

```diff
@@ -12,7 +12,8 @@
            },
            name: 'CCID'
 
-  provider :developer unless Rails.env.production?
+  provider :developer unless (Rails.env.production? ||
+                              ENV['ENABLE_TMP_LOGINS'].present?)
```

Comment:
You are right unless gets very confusing, I will switch it to if statement using `if !Rails.env.production? ||  ENV['ENABLE_TMP_LOGINS'].present?` or maybe `if Rails.env.development? ||  ENV['ENABLE_TMP_LOGINS'].present?`